.github/workflows/test.yml

```
@@ -0,0 +1,20 @@
+name: Test
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [10, 12, 14]
```

```suggestion
        node: [14]
```

This is just a starter repo, so we don't have an official support policy and testing node LTS is sufficient.

---

Updated in bd2db3b